lib/web_ui/dev/roll_fallback_fonts.dart

```diff
@@ -307,14 +311,15 @@ OTHER DEALINGS IN THE FONT SOFTWARE.
     ]);
   }
 
-  Future<void> _addApiFallbackFonts(
+  Future<List<String>> _processFallbackFonts(
     http.Client client,
-    List<String> fallbackFonts,
+    List<String> availableFonts,
```

Comment:
If you still want to move forward with this change (no objection from me, the other comment was just my personal preference), then let's rename this to `requestedFonts` or similar.

Reply:
Yep, `requestedFonts` sounds good to me!